File: Backend/src/main/java/com/feedle/feedleapi/Networking/MakeFriendRequest.java
package com.feedle.feedleapi.Networking;

import com.feedle.feedleapi.Models.FriendRequestNotification;

public class MakeFriendRequest extends  Request {
    private FriendRequestNotification friendRequestNotification;
    public MakeFriendRequest(FriendRequestNotification friendRequestNotification)
    {
        super(RequestType.MakeFriendRequest);
        this.friendRequestNotification = friendRequestNotification;
    }

    public FriendRequestNotification getFriendRequestNotification() {
        return friendRequestNotification;
    }
}

File: FeedleDataTier/FeedleDataTier/Network/MakeFriendRequest.cs
using System.Text.Json.Serialization;
using FeedleDataTier.Models;

namespace FeedleDataTier.Network
{
    public class MakeFriendRequest : Request
    {
        [JsonPropertyName("friendRequestNotification")] 
        public FriendRequestNotification FriendRequestNotification { get; set; }
        public MakeFriendRequest(FriendRequestNotification friendRequestNotification) : base(RequestType.MakeFriendRequest)
        {
            FriendRequestNotification = friendRequestNotification;
        }
    }
}